unixfs/hamt/util.go

```diff
@@ -2,6 +2,7 @@ package hamt
 
 import (
 	"math/big"
+	"math/bits"
```

Comment:
If we are adding this we should increase checked Go version to 1.9 in package.json and Makefiles.